core/src/components/checkbox/checkbox.md.vars.scss

```diff
@@ -17,7 +17,7 @@ $checkbox-md-background-color-focused:   ion-color(primary, tint) !default;
 $checkbox-md-icon-background-color-on:   ion-color(primary, base) !default;
 
 /// @prop - Size of the checkbox icon
-$checkbox-md-icon-size:                  dynamic-font(18px) !default;
+$checkbox-md-icon-size:                  18px !default;
```

Comment:
Can we add a code comment that explains why we do not use dynamic-font here?

Reply:
done